src/skmultiflow/evaluation/evaluate_prequential_delayed.py

```diff
@@ -0,0 +1,518 @@
+import os
+import warnings
+import re
+import numpy as np
+import pandas as pd
+from timeit import default_timer as timer
+
+from numpy import unique
+
+from skmultiflow.evaluation.base_evaluator import StreamEvaluator
+from skmultiflow.utils import constants
+from skmultiflow.utils import TimeManager
+
+
+class EvaluatePrequentialDelayed(StreamEvaluator):
+    """ The prequential evaluation method or interleaved test-then-train method.
```

Comment:
This documentation needs to be updated accordingly.

Reply:
Updated in commit [28d49da84c2291bb5bb0329492fafefa769cc3bc](https://github.com/scikit-multiflow/scikit-multiflow/commit/28d49da84c2291bb5bb0329492fafefa769cc3bc). Not sure if it is good enough.

Reply:
Looks good to me :-) Thanks!